meta/main.yml

```diff
@@ -0,0 +1,130 @@
+---
+galaxy_info:
```

Comment:
This whole section probably needs completing appropriately if it's going to get submitted to Galaxy, or perhaps removed if it's not.